resolwe/process/runtime.py

```diff
@@ -8,6 +9,15 @@
 
 try:
     from plumbum import local as Cmd  # pylint: disable=unused-import
+    # Log plumbum commands to standard output.
+    root = logging.getLogger('plumbum.local')
+    root.setLevel(logging.DEBUG)
+
+    handler = logging.StreamHandler(sys.stdout)
+    handler.setLevel(logging.DEBUG)
+    formatter = logging.Formatter('%(asctime)s - %(message)s')
+    handler.setFormatter(formatter)
+    root.addHandler(handler)
```

Comment:
Group the configurations together, i.e. first all `formatter` configuration, then `handler` and `root` at the end.